app/src/services/imageService.js

```diff
@@ -10,13 +10,9 @@ const AWS = require('aws-sdk');
 
 const geoQuery = require('services/imageService/geoQuery.json');
 const viirsTemplate = require('services/imageService/template/viirs.json');
-const formaAlertsTemplate = require('services/imageService/template/forma-alerts.json');
-const imazonAlertsTemplate = require('services/imageService/template/imazon-alerts.json');
 
 const LAYERS_PARAMS_MAP = {
     'viirs-active-fires': viirsTemplate,
```

Comment:
If there is only one key is this still needed?

Reply:
Turns out this whole class (ImageService) is no longer needed with the removal of these alerts, so I removed this service altogether :partying_face: